praw/models/reddit/subreddit.py

```diff
@@ -480,7 +480,7 @@ def subscribe(self, other_subreddits=None):
             list of subreddits.
 
         """
-        data = {'action': 'sub', 'skip_inital_defaults': True,
```

Comment:
Oh no! I wonder if this defaulted to True, or does this change the behavior? If it does change the behavior that will technically be a breaking change and we'll have to put it into PRAW5.